src/votekit/metrics/distances.py

```diff
@@ -0,0 +1,108 @@
+from votekit.profile import PreferenceProfile
+from votekit.graphs.models import BallotGraph
+import numpy as np
+import ot  # type: ignore
+import networkx as nx  # type: ignore
+from typing import Union, Optional
+
+
+def earth_mover_dist(pp1: PreferenceProfile, pp2: PreferenceProfile) -> int:
+    """
+    Computes the earth mover distance between two elections. \n
+    Assumes both elections share the same candidates
+    """
+    # create ballot graph
+    graph = BallotGraph(source=pp2, complete=True)
+    ballot_graph = graph.from_profile(profile=pp2, complete=True)
```

Comment:
if the source for the Graph is a preference profile it updates the weights, so this line is redundant

Reply:
done